tools/shared.py

```diff
@@ -2230,6 +2230,9 @@ def get_safe_internalize():
     internalize_public_api = '-internalize-public-api-'
     internalize_list = ','.join([exp[1:] for exp in exps])
 
+    if Settings.FETCH:
+      internalize_list += ",emscripten_fetch"
```

Comment:
adding it to the `exps` list a few lines earlier would be safer (as in theory if the list is empty, starting it with a comma is invalid).

however, I think a better solution is to add that function to EXPORTED_FUNCTIONS in emcc.py (around the existing fetch code there, probably). or is there a reason that can' work?